deepspeed/runtime/engine.py

```diff
@@ -1470,7 +1470,7 @@ def _configure_fp16_optimizer(self, optimizer):
                     has_moe_layers=self.has_moe_layers,
                 )
         else:
-            log_dist("Creating fp16 unfused optimizer with dynamic loss scale",
+            log_dist(f'Creating {model_dtype} unfused optimizer with dynamic loss scale',
```

Comment:
I would have the same question for this message since the only optimizer that is initialized after is `optimizer = FP16_UnfusedOptimizer`